client/account-recovery/lost-password/lost-password-form/style.scss

```diff
@@ -0,0 +1,22 @@
+.lost-password-form__title {
+	font-size: 1.5em;
+	font-weight: bold;
```

Comment:
@mtias Do you think it'd be a good idea to add `$bold: 600` as a variable to `_typography.scss` to simplify this and have it be a single point of updating if this ever changes? Usage being `font-weight: $bold;` which I think reads nicely too.